src/scenes/Activities/Favorite/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component, Fragment } from 'react';
+import { inject, observer } from 'mobx-react';
+
+import theme from '../../../config/theme';
+import InfiniteScroll from '../../../components/InfiniteScroll';
+import EventCard from '../../../components/EventCard';
+import Loading from '../../../components/EventListLoading';
+
+
+@inject('store')
+@observer
+export default class Favorite extends Component {
+  componentDidMount() {
+    this.props.store.favorite.init();
+  }
+
+  render() {
+    const { displayList, loadingMore, loading } = this.props.store.favorite;
+    if (loading) return <Loading />;
+    const events = (displayList || []).map((event, i) => <EventCard key={i} index={i} event={event} />); // eslint-disable-line
+    return (
+      <Fragment>
+        <InfiniteScroll
```

Comment:
dont use `InfiniteScroll` if its not using the loadMore functionality

Reply:
Yes agreed. We should have an `EventCardGrid` component and the `InfiniteScroll` can use `EventCardGrid` within it as well.